README.md

```
@@ -1,521 +1,307 @@
+    C --> D[FraudDetectionEnsemble]
+    D --> E{Prediction}
+    E --> F[API /predict]
+    E --> G[Backtest CLI]
+    D --> H[ModelMonitor]
+    H --> I[Drift Alerts]
 ```
 
-### 📁 Estrutura do Projeto
+### Project Structure
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)